### DISCUSSION TITLE
Performance insights, queueing?

### DISCUSSION THREAD

**Discussion by ChrisHardie** [spatie/laravel-stats#8] [General]
Thanks for this awesome package! 

Based on your usage so far, do you have any insights around performance, i.e. is it safe to use a call like `increase()` during a user-facing request without slowing things down? Is there a case to be made for queuing these operations to take place in the background? Thank you.

**Comment by freekmurze** [comment]
`increase()` only inserts a record in a table. I'm thinking in many cases it shouldn't pose any performance problems.

**Reply by HassanZahirnia** [reply]
@freekmurze I also had same performance concerns, I noticed the migration published by the package does not include any indexes.
Is that by design or it's something not needed ?

Because I think this table will end up with thousands/millions of rows assuming someone would want to keep stats of multiple models ( triggering `increase` and `decrease` inside model `created` and `deleted` events )
